packages/organizations/origin-organization-irec-api-client/src/generateSchema.ts

```diff
@@ -17,7 +17,7 @@ export const generateSchema = async () => {
                 username: process.env.DB_USERNAME ?? 'postgres',
                 password: process.env.DB_PASSWORD ?? 'postgres',
                 database: process.env.DB_DATABASE ?? 'origin',
-                entities,
+                entities: [...entities],
```

Comment:
Maybe I missed something. How is `[...entities]` different from just `entities`?

Reply:
Yep, old artefact. I'll remove it